aerosol_dynamics/particle_test.py

```diff
@@ -46,26 +69,48 @@ def test_knudsen_number():
 
 
 def test_slip_correction_factor():
+    """"
+    Test that the slip correction factor is calculated correctly.
+    TODO:
+    - can we check units instead of numbers?
+    - that's an overly specific number with lots of digs
```

Comment:
That's fine, we can pare it back with sigfigs. These numbers were taken directly from what the code output. I was ensuring that the numbers were not changing as the file was evolving.

Reply:
Aaaahhhh! Now I understand where these raw numbers come from. Okay, let me think more